test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -1354,6 +1355,9 @@ func newV1beta1WebhookHandler(t *testing.T, holder *holder, phase string, conver
 		if err := json.NewEncoder(w).Encode(review); err != nil {
 			t.Errorf("Marshal of response failed with error: %v", err)
 		}
+		// TODO: temporary hold the request to avoid TCP collisions on the client
+		// xref: https://github.com/kubernetes/kubernetes/issues/109022#issuecomment-1083362211
+		time.Sleep(50 * time.Millisecond)
```

Comment:
Is there anything we can potentially wait for instead of sleeping?

[Also - 50ms is really small in our overloaded CI env - if we really need sleep, we should consider something higher.]

Reply:
I'm afraid that depends on having more info from https://github.com/golang/go/issues/50984 ,